doc/hooks/automake.section.md

```diff
@@ -0,0 +1,4 @@
+
+### Automake {#setup-hook-automake}
+
+Adds the `share/aclocal` subdirectory of each build input to the `ACLOCAL_PATH` environment variable.
```

Comment:
Short files like this one are necessary for doc tooling reasons.

Instead of worrying about short files, we should be adding examples, explanations, etc, or move the content to other chapters and perhaps index them. That's for later though.